routes/index.js

```diff
@@ -1,14 +1,13 @@
 var express = require('express');
 var router = express.Router();
```

Comment:
Logger seems to be unused. Please remove.

Reply:
sf
sdf
df

Reply:
**this is a great PR**